### ISSUE TITLE
Install script ignores JSON_PATH (upgrading from old script)

### ISSUE THREAD

**Issue by henrypijames** [v2fly/fhs-install-v2ray#109]
I followed the Wiki instructions to upgrade from old script to new script, and changed `JSON_PATH` after downloading the new script and before running it. However, the change had no effect, and starting up v2ray via systemctl resulted in error:

```
panic: open /usr/local/etc/v2ray/config.json: no such file or dicrectory
```

Looking at `/etc/systemd/system/v2ray.service` confirms that `ExecStart` carries the default path and ignores `JSON_PATH` from the install script.

Update: It looks like the default path is hard-coded in `v2ray.service`, and the install script needs to change it to match `JSON_PATH` after extracting it from the install package.

**Comment by IceCodeNew** [comment]
This issue should really be addressed by this project. But unfortunately, I'm not available for the following days due to my new term is about to start.
You can fix this problem by creating a drop-in configuration, they should be like follows.

```
export JSON_PATH='/anywhere_you_want'
mkdir -p '/etc/systemd/system/v2ray.service.d'
echo "ExecStart=
ExecStart=/usr/local/bin/v2ray -confdir $JSON_PATH" \
> '/etc/systemd/system/v2ray.service.d/v2ray_execstart_custom_json_path.conf'
systemctl daemon-reload && systemctl restart v2ray
```

**Comment by IceCodeNew** [comment]
@henrypijames Hey, could you confirm whether PR #114 solve your problem or not?

**Comment by IceCodeNew** [comment]
Closed by #114.
